apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -185,4 +185,7 @@
 
     public static final OfficialComponent THRIFT_CLIENT = new OfficialComponent(101, "thrift-client");
 
+    public static final OfficialComponent ASYNC_HTTP_CLIENT = new OfficialComponent(102, "AsyncHttpClient");
```

Comment:
This one should be added to the `skywalking/oap-server/server-bootstrap/src/main/resources/component-libraries.yml`

Reply:
OK,thanks.